server/controller/signupController.js

```diff
@@ -0,0 +1,35 @@
+import pool from '../config/configDb';
+import { getSignupQuery } from '../models/Queries';
+
+const { User } = require('../models/User');
+
+function signup(req, res) {
+  const {
+    email, password, fname, lname, address, country,
+  } = req.body;
+  const user = new User(0, email, fname,
+    lname, password, address, country, 'unverified', false);
+  const values = Object.keys(user).map(key => user[key]);
+  values.splice(1, 1);
+  pool.query(getSignupQuery(values))
+    .then((result) => {
+      const newUser = user;
+      newUser.id = result.rows[0].id;
+      return res.status(201).send({
+        status: 201,
+        data: newUser.toJSON(),
```

Comment:
The **newUser** will also contain password. You don't want to return a users password in an endpoint.

Reply:
I solved it with the toJson() method in the user class